### DISCUSSION TITLE
exception when up/down-arrow key pressed with empty search (meta-k)

### DISCUSSION THREAD

**Discussion by phrohdoh** [athensresearch/athens#827] [Archive]
On 1.0.0-beta57 (and reproduced on beta60 by @jsmorabito), opening the page create/search popup/view/? and pressing either the up- or down-arrow key without having entered anything will result in an exception.

![](https://cdn.discordapp.com/attachments/802652338619285594/821025006968242246/Screen_Shot_2021-03-15_at_9.20.04_AM.png)

https://github.com/athensresearch/athens/blob/v1.0.0-beta.60/src/cljs/athens/views/athena.cljs#L207-L230

I _suspect_ we'll want to invoke `scroll-into-view` iff `next-el` is non-`nil`.

**Comment by phrohdoh** [comment]
I've shared a solution at https://discord.com/channels/708122962422792194/800439820190220339/821044549145198622 for review. This can of course become a PR if desired.

**Reply by jefftangx** [reply]
> I suspect we'll want to invoke scroll-into-view iff next-el is non-nil.

Correct. Create a PR!
